### PR TITLE
Allow custom dnsmasq_router when set as string.

### PR DESCRIPTION
**Pull request by ypid**
* The previous behavior was not flexible enough as it only allowed to
  announce the dnsmasq system as default gateway or to not announce any
  default route via DHCPv4.

* Maybe use another variable for this?

### CODE REVIEW

**Comment by drybjed** [comment]
Yes, I would go with `dnsmasq_router_address`, make that empty by default (`''`). I tried using variables as both bools and data but in the end it brings more problems than it solves. Otherwise, nice idea, thanks.

**Comment by drybjed** [comment]
@ypid I need to update the `.travis.yml` file in this role to request `sudo`, you will need to rebase this PR after that.

**Comment by ypid** [comment]
Sure, I will do that. Just fixed the variable naming. Commit message:

Use dnsmasq_interfaces[i].gateway to specify custom gw instead of dnsmasq_router.

* After discussing this feature with @drybjed and renaming the variable
  I noted that a global variable for the gw which is subnetwork
  dependent does not make much sense.
  https://github.com/debops/ansible-dnsmasq/pull/15

**Comment by drybjed** [comment]
@ypid OK, try rebasing this PR, it should work now on the old Travis hosts.

File: defaults/main.yml [hunk 1]
```diff
@@ -25,7 +27,7 @@ dnsmasq_bind_interfaces: True
 # Block private IP addresses coming from upstream nameservers
 dnsmasq_stop_dns_rebind: True
 
-# Don't forwars reverse DNS lookups for private IP addresses
+# Don't forwards reverse DNS lookups for private IP addresses
```
Comment (drybjed) [inline defaults/main.yml]: Should probably be `forward`. :-)

**Pull request merged**